classes/options-general.php

```diff
@@ -328,51 +328,51 @@ function print_page() {
 
 													<tr valign="top">
 														<td width="150"><label
-																for="address_format"><?php _e( 'Address Format', 'simplemap' ); ?></label>
+																for="address_format"><?php _e( 'Address Format', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 138 characters

<!-- Comment by [SideCI](https://sideci.com) -->